test/scalability/default_generator_config.yaml

```diff
@@ -0,0 +1,31 @@
+- className: cohort
+  count: 5
+  queuesSets:
+  - className: cq
+    count: 6
+    nominalQuota: 20
+    borrowingLimit: 100
+    reclaimWithinCohort: Any
+    withinClusterQueue: LowerPriority
+    workloadsSets:
+    - count: 350
+      msBetweenWlCreation: 100
+      workloads:
+      - className: small
+        msRunning: 200 # will eventually queue
+        priority: 50
+        request: 1
+    - count: 100
+      msBetweenWlCreation: 500
+      workloads:
+      - className: medium
+        msRunning: 500 # will eventually be evicted
+        priority: 100
+        request: 5
+    - count: 50
+      msBetweenWlCreation: 1200
+      workloads:
+      - className: large
+        msRunning: 1000
+        priority: 200
+        request: 15 # leave some space for a medium to run
```

Comment:
So why 15? We would like to simulate the case of workloads eating the entire quota for CQ, because apparently this is what some of our users would do.